barretenberg/cpp/src/barretenberg/translator_vm/goblin_translator_flavor.hpp

```diff
@@ -287,13 +277,22 @@ class GoblinTranslatorFlavor {
                               relation_wide_limbs_range_constraint_0,       // column 76
                               relation_wide_limbs_range_constraint_1,       // column 77
                               relation_wide_limbs_range_constraint_2,       // column 78
-                              relation_wide_limbs_range_constraint_3,       // column 79
-                              ordered_range_constraints_0,                  // column 80
-                              ordered_range_constraints_1,                  // column 81
-                              ordered_range_constraints_2,                  // column 82
-                              ordered_range_constraints_3,                  // column 83
-                              ordered_range_constraints_4);                 // column 84
+                              relation_wide_limbs_range_constraint_3);      // column 79
+    };
+
+    // TODO(https://github.com/AztecProtocol/barretenberg/issues/907)
+    // Note: These are technically derived from wires but do not depend on challenges (like z_perm). They are committed
+    // to in the wires commitment round.
+    template <typename DataType> class OrderedRangeConstraints {
```

Comment:
Talked to Kesha about this before I made the change. Basically these polys are not quite wires but not quite derived entities so they need their own category for now. This allowed me to get rid of the big hard coded initializations in the prover.